modules/mediawiki/files/bin/mwscript.py

```diff
@@ -2,9 +2,12 @@
 # -*- coding: utf-8 -*-
 import sys
 import os
+long = False
 if len(sys.argv) < 3:
     raise Exception("Not Enough Parameters")
 script = sys.argv[1]
+if script == 'importDump.php':
```

Comment:
I understand this isn't the final version of this, but this is probably better:
```py
longScripts = ("deleteBatch.php", "importDump.php", "importImages.php", "etc...")
if script in longScripts:
	long = True
```

Reply:
Yes, needed to think about which scripts first would be in the list